Fixed tk window display issue and added countdown timer

paperboi opened the pull request:
Fixes #1 
- Window generated during break time now draws over other apps
- Text formatting for time (now in HH:MM format)
Added
- Countdown timer to break

paperboi committed:
Fixed tk window display issue and added countdown timer

Fixed
- Window generated during break time now draws over other apps
- Text formatting for time (now in HH:MM format)
Added
- Countdown timer to break
-

autolyticus committed:
Update launch.cmd

autolyticus commented on tab.py:
```diff
@@ -14,32 +14,27 @@
     print('Note: Using default Pomodoro work duration of 25 minutes')
 else:
-    workMins = float(sys.argv[1])
+    workMins = int(sys.argv[1])
```
Any specific reason you're changing the inputs to int?

paperboi replied:
Changed them to int for the countdown function. Zeroes after decimal point were being displayed too. Changing it to int was more of a workaround solution there.

autolyticus replied:
I would recommend keeping the float, it gives more flexibility

autolyticus replied:
You can use another format string to prevent the decimal points from being displayed if necessary

autolyticus commented on utils.py:
```diff
@@ -48,6 +49,18 @@ def waitForUserActivity():
+    while t:
+        mins, secs = divmod(t, 60)
+        timeformat = '{:02d}:{:02d}'.format(mins, secs)
+        print(f"Next break in {timeformat}", end="\r")
```
If you added a sendNotification here, maybe you could have an updating notification every second, did you try that out?

paperboi replied:
Yes. It's not possible - overwriting messages is not supported by sendNotification

paperboi committed:
Modified launch.cmd, tab.py, utils.py

Changed launch cmd args, workMins & breakMins var types, time formatting in countdown

paperboi commented on tab.py:
```diff
@@ -9,32 +9,32 @@
         breakEndTime = datetime.now() + timedelta(minutes=breakMins)
-        print(f'Break until {breakEndTime.hour}:{breakEndTime.minute}')
+        # print(f'Break: {breakEndTime.strftime("%H:%M")}')
+        print(f'Break until {breakEndTime.strftime("%H:%M")}')
```
Changed time formatting to datetime.now().strftime

paperboi commented on tab.py:
```diff
@@ -9,32 +9,32 @@
+        print(f'Break until {breakEndTime.strftime("%H:%M")}')
         sendNotification(
-            f'You are obliged to take a break. See you at {breakEndTime.strftime("%H:%M")}',
+            f'You are obliged to take a break. See you at {breakEndTime.strftime("%H:%M")}'
```
Changed time formatting to datetime.now().strftime

paperboi commented on utils.py:
```diff
@@ -52,7 +52,7 @@ def waitForUserActivity():
     while t:
         mins, secs = divmod(t, 60)
-        timeformat = '{:02d}:{:02d}'.format(mins, secs)
+        timeformat = '{:02.0f}:{:02.0f}'.format(mins, secs)
```
Formatted string output to reflect workMins, breakMins' type change from int to float

reisub0 commented:
Thanks for the pull request @paperboi. I hope you are finding this useful.

Cheers! :clinking_glasses:
